backend/services/task.service.js

```
@@ -0,0 +1,785 @@
+    if (!project) {
+      throw new Error("Project not found for task");
+    }
+
+    const git = githubToken ? new GitService(githubToken) : this.gitService;
+
+    switch (operation) {
+      case "toBase":
+        // First ensure the task branch is pushed
+        const pushResult = await git.push(task.worktree_path, task.branch, {
+          setUpstream: true,
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)